tests/test_forms.py

```diff
@@ -0,0 +1,137 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from taxi.models import Manufacturer, Car
+from taxi.forms import (
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 

Read more about topic [here](https://peps.python.org/pep-0008/#imports)